add-on/chrome/bootstrap.js

```diff
@@ -29,6 +29,9 @@ XPCOMUtils.defineLazyModuleGetter(this, "Task",
  * to provide the required loop functions for the window.
  */
 var WindowListener = {
+  // Records the add-on version once we know it.
+  addonVersion: "unknown",
```

Comment:
I generally prefer to avoid global variables, and tack them to where they are used. MozLoopService is different due to the frozen interfaces.